src/apis/lpn.ts

```diff
@@ -12,7 +12,7 @@ export function getContainerDetail(id: string) {
   return apiClient.get(`/containers/${id}/details`);
 }
 
-export function getStatusDetails(id: string) {
-    return apiClient.post(`/containers/${id}`,{});
+export function getStatusDetails(id: string,status:any) {
```

Comment:
Rename it to `updateContainerStatus`. This naming suggests, that you are getting data instead of posing it.

Reply:
Add space after the comma.